src/Assets/TestProjects/NetCoreCsharpAppReferenceCppCliLib/CSConsoleApp/CSConsoleApp.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>$(CurrentTargetFramework)</TargetFramework>
+    <TargetFramework>$(CurrentTargetFramework)-windows</TargetFramework>
```

Comment:
We have to take this change - the CI runners changed configuration and now include a version of the C++ tooling that requires building for windows - this hit a bunch of other PRs, but hadn't hit the 7.0.300 branch because we don't have any active codeflow to this branch. This PR was just the unlucky one that triggered a build showing the issue :)

I've taken the change directly from the fix commits on the other branches, so I could make that case to tactics if necessary.